component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java

```diff
@@ -758,6 +761,12 @@ public void fileSinkTest() throws InterruptedException {
 
         Thread.sleep(1000);
         siddhiAppRuntime.shutdown();
+        File sinkRoot = new File(sinkUri);
+        try {
+            FileUtils.deleteDirectory(sinkRoot);
+        } catch (IOException e) {
+            throw new TestException("Failed to delete files in due to " + e.getMessage(), e);
```

Comment:
Better log the sinkUri too.

Reply:
Fixed in cdb7b86019f370bcb90c545043b2ad205e8e98df